src/test/resources/featurefiles/NavigateValoriWebsite.feature

```diff
@@ -1,13 +1,16 @@
 Feature: Test navigation on the valori.nl homepage
 
+  @Runtag
```

Comment:
Deze tags gebruikte je om lokaal te runnen maar het is beter om deze niet te committen naar de repo.